data/remnant missions.txt

```diff
@@ -1252,3 +1252,214 @@ mission "Remnant: Expanded Horizons Quarg 3"
 			`	"Our historical records just have a few pictures and videos of Quarg ships, along with a basic scan that doesn't provide more than the general outline of the ship. These scans," he says while tapping the databanks, "actually picked up detailed structural layouts and even identified a number of key components. It isn't nearly enough information to be able to duplicate them, but it is enough that we can start trying to figure out how they work.`
 			`	Upon arriving at <planet> they quickly thank you, hand you <payment>, and run back to their offices to finish prepping their material to publish and distribute.`
 
+
+
+mission "Remnant: Broken Jump Drive 1"
+	name "Jump Drive (Broken)"
+	minor
+	invisible
+	landing
+	to offer
+		has "remnant blood test pure"
+	to complete
+		has "Jump Drive (Broken)"
```

Comment:
`has "Jump Drive (Broken)"` is parsed by the game into the `ConditionSet` with the expression `"Jump Drive (Broken)" != 0`

Are you meaning that this mission should not complete until the player is in possession of the `Outfit` with the name `"Jump Drive (Broken)"`?

Reply:
Basically, yes. This is an attempt to have the mission actually complete before the player returns to the planet and triggers the following mission. 

Ideally, I'd just have the mission finished as "done" immediately after the player accepts to sell the broken jump drive to them, because there is no reason why the mission should need to continue after they've completed the transaction, but apparently that is not a thing we can do, so the next best seems to be either have it complete as soon as the player acquires another broken jump drive, or figure out how to have it complete as soon as they take off.

Reply:
@zitchas note that in the "on accept" `MissionAction` you fail this mission, so it can't / won't ever execute its "on complete" clause. You don't need a "to complete" `ConditionSet` for missions that won't complete due to other factors. Since it's invisible as well, the player won't see it when viewing the `MissionPanel`